subs/optimal_737.py

```diff
@@ -86,6 +86,7 @@ def get737_optimal_subs():
             '\\rho_{fuel}': 817.*units('kg/m^3'),  # Kerosene [TASOPT]
             'f_{wingfuel}': 1,  #.35 may be a better answer
             'TipReduct': 1,
+            '\\tau_{max_w}': 0.1267,
```

Comment:
@mayork I'm not sure I understand what you mean by you "went to a common wing box model". Wasn't that already implemented? I'm confused by the PR too. Is there a parallel effort to this branch?